bip-schnorr.mediawiki

```diff
@@ -140,9 +140,9 @@ All provided signatures are valid with overwhelming probability if and only if t
 * Generate ''u-1'' random integers ''a<sub>2...u</sub>'' in the range ''1...n-1''. They are generated deterministically using a [https://en.wikipedia.org/wiki/Cryptographically_secure_pseudorandom_number_generator CSPRNG] seeded by a cryptographic hash of all inputs of the algorithm, i.e. ''seed = seed_hash(pk<sub>1</sub>..pk<sub>u</sub> || m<sub>1</sub>..m<sub>u</sub> || sig<sub>1</sub>..sig<sub>u</sub> )''. A safe choice is to instantiate ''seed_hash'' with SHA256 and use [https://tools.ietf.org/html/rfc8439 ChaCha20] with key ''seed'' as a CSPRNG to generate 256-bit integers, skipping integers not in the range ''1...n-1''.
 * For ''i = 1 .. u'':
 ** Let ''P<sub>i</sub> = point(pk<sub>i</sub>)''; fail if ''point(pk<sub>i</sub>)'' fails.
-** Let ''r = int(sig<sub>i</sub>[0:32])''; fail if ''r &ge; p''.
+** Let ''r = sig<sub>i</sub>[0:32]''; fail if ''int(r) &ge; p''.
 ** Let ''s<sub>i</sub> = int(sig<sub>i</sub>[32:64])''; fail if ''s<sub>i</sub> &ge; n''.
-** Let ''e<sub>i</sub> = int(hash(bytes(r) || pk<sub>i</sub> || m<sub>i</sub>)) mod n''.
+** Let ''e<sub>i</sub> = int(hash(r || pk<sub>i</sub> || m<sub>i</sub>)) mod n''.
 ** Let ''R<sub>i</sub> = lift_x(r)''; fail if ''lift_x(r)'' fails.
```

Comment:
In commit "leave r as byte array":

If r is a byte array this needs to be `bytes(r)` rather than `lift_x(r)`.